src/cloud-resources/src/crd/gen/vpcendpoints.json

```diff
@@ -0,0 +1 @@
+{"apiVersion":"apiextensions.k8s.io/v1","kind":"CustomResourceDefinition","metadata":{"name":"vpcendpoints.materialize.cloud"},"spec":{"group":"materialize.cloud","names":{"categories":[],"kind":"VpcEndpoint","plural":"vpcendpoints","shortNames":["vpce"],"singular":"vpcendpoint"},"scope":"Namespaced","versions":[{"additionalPrinterColumns":[{"description":"Name of the VPC Endpoint Service to connect to.","jsonPath":".spec.awsServiceName","name":"AwsServiceName","priority":1,"type":"string"},{"description":"Availability Zone IDs","jsonPath":".spec.availabilityZoneIds","name":"AvailabilityZoneIDs","priority":1,"type":"string"}],"name":"v1","schema":{"openAPIV3Schema":{"description":"Auto-generated derived type for VpcEndpointSpec via `CustomResource`","properties":{"spec":{"properties":{"availabilityZoneIds":{"items":{"type":"string"},"type":"array"},"awsServiceName":{"type":"string"}},"required":["availabilityZoneIds","awsServiceName"],"type":"object"},"status":{"nullable":true,"type":"object"}},"required":["spec"],"title":"VpcEndpoint","type":"object"}},"served":true,"storage":true,"subresources":{"status":{}}}]}}
```

Comment:
This is the generated CRD definition which is derived from the spec in `src/cloud-resources/src/crd/vpc_endpoints.rs`. It is used in `cloudtest` to register the CRD, since the environment-controller (which would normally register the CRD) doesn't exist in `cloudtest`.

We enforce that it matches the rust code with a cargo test.